src/components/Map.js

```diff
@@ -19,17 +20,36 @@ import {
   DateTime
 } from "../api_client/index";
 import { BoundingBox, isSamePoint } from "../util/bounds";
-
+import logo_lizard from "../graphics/logo-Lizard.png";
+import L from "leaflet";
```

Comment:
A small question: do we also need leaflet if we already have react-leaflet?

Reply:
In this case it is not clear if react-leaflet exposes the utility functions:
 -Leaflet.point()
-Leaflet.icon()
I could not find them in the docs. 
https://react-leaflet.js.org/docs/en/components.html
Therefore I used these functions directly from leaflet.
I used klimaatatlas as an example.